app/views/unattended/report_templates/host_-_applicable_errata.erb

```diff
@@ -22,7 +22,7 @@ require:
 ￼ version: 3.16.0
 -%>
 <%- report_headers 'Host', 'Operating System', 'Environment', 'Erratum', 'Type', 'Published', 'Applicable since', 'Severity', 'Packages', 'CVEs', 'Reboot suggested' -%>
-<%- load_hosts(search: input('Hosts filter'), includes: [:operatingsystem, :applicable_errata, :lifecycle_environment]).each_record do |host| -%>
```

Comment:
Why did you drop the `applicable_errata`? wouldn't this cause n+1 queries in `host_applicable_errata_filtered`?

Reply:
Well no, because the `host_applicable_errata_filtered` causes it anyway.
If you apply any additional where conditions to preloaded data, it doesn't use them and performs the query anyway, so we were just loading useless data.

If we would want to leverage the preload of `applicable_errata`, we would need to drop the Errata filter.
That would be major speed improv woulent, but it would cause reduced functionality.

I've tried loading the filtered errata's ids and then just filter them in memory, but that was even worse, if there is a large errata dataset per host, it takes more time to filter them in memory then fetching them from DB.